ocaml/xapi/xapi_vusb.ml

```diff
@@ -34,6 +34,13 @@ let create ~__context ~vM ~uSB_group ~other_config =
         another vusb can not create with the same USB_group. *)
     if vusbs <> [] then
       raise (Api_errors.Server_error(Api_errors.usb_group_conflict, [Ref.string_of uSB_group]));
+
+    (* USBs on different host can not be attached to one vm.*)
+    let possible_hosts = Xapi_vm.get_possible_hosts ~__context ~vm:vM in
+    let pusbs = Db.USB_group.get_PUSBs ~__context ~self:uSB_group in
+    let host = Db.PUSB.get_host ~__context ~self:(List.hd pusbs) in
+    if not (List.mem host possible_hosts) then
```

Comment:
if you really want to fulfil the purpose of the comments, you have to:
1. get the list of host from the pusbs;
2. judge if they are the same one;
3. determine if it (the same host from the pusbs) is in the possible host of the VM.

Reply:
I think no need to judge the list of pusbs , as all the pusbs must be on the same host. It is enough to judge one of them. Also now USB_group will only have one PUSB

Reply:
If that is the case, then fine. But it is better to add some comments saying so, otherwise the code will look odd; the USB_group looks like a group of some USBs, and though it is one PUSB for now, it might be used for extension in the future (think of you may want to have a group of USB device of similar kind of functionality on different hosts, so you can do VM migrate or so), so do not put mine in that.